cmd/kogito/command/install/kogito_operator.go

```diff
@@ -78,19 +78,19 @@ func (i *installKogitoOperatorCommand) RegisterHook() {
 
 func (i *installKogitoOperatorCommand) InitHook() {
 	i.flags = installKogitoOperatorFlags{
-		OperatorFlags: common.OperatorFlags{},
+		OperatorFlags: flag.OperatorFlags{},
 	}
 	i.Parent.AddCommand(i.command)
-	common.AddOperatorFlags(i.command, &i.flags.OperatorFlags)
+	flag.AddOperatorFlags(i.command, &i.flags.OperatorFlags)
 
 	i.command.Flags().StringVarP(&i.flags.namespace, "project", "p", "", "The project name where the operator will be deployed")
 	i.command.Flags().StringVarP(&i.flags.image, "image", "i", shared.DefaultOperatorImageNameTag, "The operator image")
 	i.command.Flags().BoolVar(&i.flags.installDataIndex, "install-data-index", false, "Installs the default instance of Data Index being provisioned by the Kogito Operator in the project")
 	i.command.Flags().BoolVar(&i.flags.installJobsService, "install-jobs-service", false, "Installs the default instance of Jobs Service being provisioned by the Kogito Operator in the project")
 	i.command.Flags().BoolVar(&i.flags.installMgmtConsole, "install-mgmt-console", false, "Installs the default instance of Management Console being provisioned by the Kogito Operator in the project")
 	i.command.Flags().BoolVar(&i.flags.installAllServices, "install-all-services", false, "Installs the default instance of every Kogito Support services (Data Index, Jobs Service, etc.) being provisioned by the Kogito Operator in the project")
-	i.command.Flags().BoolVarP(&i.flags.force, "force", "f", false, "When set, the operator will be installed in the current namespace using a custom image, e.g. quay.io/kiegroup/kogito-cloud-operator:my-custom-tag")
-	i.command.Flags().BoolVar(&i.flags.enablePersistence, "enable-persistence", false, "If set will install Infinispan in the same namespace and inject the environment variables to configure the service connection to the Infinispan server.")
+	i.command.Flags().BoolVarP(&i.flags.force, "force", "f", false, "When set, the operator will be installed in the current project using a custom image, e.g. quay.io/kiegroup/kogito-cloud-operator:my-custom-tag")
```

Comment:
Project is OpenShift term, Kubernetes use only `namespace`.
Wouldn't it have more sense to keep `namespace` wording here?

Reply:
done